<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- gives static world tf -->
    <node pkg="tf" type="static_transform_publisher" name="base_link_broadcaster" args="0 0 0 0 0 0 world icp 100" />

    <!-- view rviz for visualizations -->
    <node pkg="rviz" type="rviz" name="icp_matcher_viewer"/>

    <!-- launch icp_matcher -->
    <node pkg="fetchit_icp" type="icp_matcher_node" name="icp_matcher_node" output="screen">
        <param name="template_file" value="corner.pcd"/>
        <param name="test_file" value="2.pcd"/>
        <param name="max_distance" value="0.5"/>
        <param name="iterations" value="1000000"/>
        <param name="trans_epsilon" value="0.000000000001"/>
        <param name="fit_epsilon" value="0.0000000000001"/>

    </node>

</launch>